src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+was acquired within the ASL time-series or as a separate scan. These and other M0 options are specified in the required M0 field. 
+It can also be stored under `fmap/sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.nii[.gz]`, 
+when the pepolar approach is used.
+
+The `*_aslcontext.tsv` lists the volume_type for each volume in a single column, in the `*_asl.nii[.gz]`, corresponding to the volume_types specified in the table below, based 
+on DICOM Tag (0018,9257) `ASL Context`. Note that the volume_types `control` and  `label` within BIDS do not discriminate between types of labeling, but only serve to specify 
+the magnetization state of the blood and thus the ASL subtraction order.
+
+| **volume_type** | **Definition**                                                                                                                                                                         |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| control         | The control image is acquired in the exact same way as the label image, except that the magnetization of the blood flowing into the imaging region has not been inverted.              |
+| label           | The label image is acquired in the exact same way as the control image, except that the blood magnetization flowing into the imaging region has been inverted.                         |
+| m0scan          | The M0 image is a calibration image, used to estimate the equilibruim magnetization of blood.                                                                                          |
+| deltam          | The deltaM image is a perfusion-weighted image, obtained by the subtraction of `control` - `label`.                                                                                    |
+| cbf             | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197)). |
+
+If the raw `control` and `label` images are not available, their derivative `deltam` should be stored within the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv` 
+instead. If the `deltam` is not available, `cbf` should be stored within the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`. When `cbf` is stored within the 
+`*_asl.nii[.gz]`, its units need to be specified in the `*_asl.json` as well.
+
+`*_aslcontext.tsv` example:
+
+| **volume_type** |
+|-----------------|
+| m0scan          |
+| control         |
+| label           |
+| control         |
+| label           |
+| control         |
+| label           |
+
+### Metadata fields applicable to both (P)CASL and PASL 
```

Comment:
```suggestion
### `*_asl.json` file

#### Metadata fields applicable to both (P)CASL and PASL 
```


Does this relates to the asl.json file?

Reply:
Yes, but this is also the case for the (P)CASL- and PASL-specific metadata fields. We wanted to clarify which fields are applicable to both LabelingTypes or only to a specific LabelingType, hence why your suggested replacement could be confusing.